src/Codeception/Test/Cest.php

```diff
@@ -95,6 +96,12 @@ public function getSpecFromMethod(): string
     public function test(): void
     {
         $actorClass = $this->getMetadata()->getCurrent('actor');
+
+        if ($actorClass === null) {
+            $message = "actor setting is missing in suite configuration. Do you still use class_name?";
```

Comment:
it may be that the question itself doesn't obviously imply that that setting was renamed. Could you add a link to the changelog or explain this change in another way please?
Likewise, the $message variable is not necessary.